storage/innobase/dict/dict0load.cc

```diff
@@ -1327,6 +1373,7 @@ dict_sys_tables_rec_read(
 
 	*flags = dict_sys_tables_type_to_tf(type, not_redundant);
 
+
```

Comment:
In InnoDB code, there never should be multiple empty rows.
Try to avoid changes to white space.